virtualization/windowscontainers/deployment/docker_windows.md

```diff
@@ -121,7 +121,7 @@ Download docker.exe from `https://aka.ms/tp4/docker` and copy it to the `windows
 Run the below command to start the docker daemon. This will need to be run each time the container host is started. This command starts the Docker daemon, specifies a virtual switch for container connectivity, and set’s the daemon to listen on port 2375 for incoming Docker requests. In this configuration Docker can be managed from a remote computer.
 
 ```powershell
-PS C:\> start-process cmd "/k docker daemon -D -b <Switch Name> -H 0.0.0.0:2375”
+PS C:\> start-process cmd "/k docker daemon -D -b <Switch Name> -H npipe:////./pipe/docker_engine -H 0.0.0.0:2375”
```

Comment:
Oh, just found "bad" typographic double quotes at the end of this line and some other places as well.